src/app/ubs-admin/components/ubs-admin-table/table-cell-select/table-cell-select.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="select_wr">
+  <div *ngIf="!isEditable">{{ currentValue | serverTranslate: lang }}</div>
+  <div *ngIf="isEditable">
+    <select (change)="chosenOption($event)" style="width: 100px; outline: none; border: none">
```

Comment:
Inline styling is bad practice, separate styles to class

Reply:
done